Spatial_Engine/Modify/Orient.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using BH.oM.Dimensional;
+using BH.oM.Geometry;
+using BH.oM.Geometry.CoordinateSystem;
+
+
+namespace BH.Engine.Spatial
+{
+    public static partial class Modify
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Orients the element from one coordinate system to another.")]
+        [Input("element", "The element to orient.")]
+        [Input("from", "The cartesian coordinate system to orient from.")]
+        [Input("to", "The cartesian coordinate system to orient to.")]
+        [Output("element", "The reoriented element.")]
+        public static IElement Orient(this IElement element, Cartesian from, Cartesian to)
```

Comment:
I am thinking if it would not be worth defaulting both `to` and `from` to the global CS? Personally I would find it handy in some circumstances

Reply:
Just read the additional comments in PR description - great minds think alike! 😃

Reply:
Done